build/depends.py

```diff
@@ -58,6 +58,13 @@ def configure(self, build, conf):
     def sources(self, build):
         return ['controllers/midi/portmidienumerator.cpp', 'controllers/midi/portmidicontroller.cpp']
 
+# TODO: how to make this optional? (see OSC in features.py)
```

Comment:
Answer: Check to see how HSS1394 does it. I believe you would move this liblo stuff to features.py, so it's only acted on if the feature is turned on.